src/AddTodoForm.js

```diff
@@ -1,20 +1,34 @@
 import React from "react";
 
 function AddTodoForm(props) {
+  const [todoTitle, setTodoTitle] = React.useState("");
+
+  const handleTitleChange = (event) => {
+    const newTodoTitle = event.target.value;
+    setTodoTitle(newTodoTitle);
+  };
+
   function handleAddTodo(event) {
     event.preventDefault();
 
-    const todoTitle = event.target.title.value;
-    props.onAddTodo(todoTitle);
+    const newTodo = {
+      title: todoTitle,
+      id: Date.now(),
+    };
+
+    props.onAddTodo(newTodo);
```

Comment:
Ok so this does work, and if it helps you to keep things in separate steps that is totally fine, but this can be written a little more cleanly like this:

```suggestion
    props.onAddTodo({
      title: todoTitle,
      id: Date.now(),
    });
```